back/lib/tasks/single_use/20250805_fix_missing_vote_term_in_graph_data_units.rake

```diff
@@ -0,0 +1,17 @@
+namespace :single_use do
+  desc 'Fix missing vote_term in graph data units, which was forgotton to be updated when introducing the vote term'
+  task :fix_missing_vote_term_in_graph_data_units, %i[execute] => [:environment] do |_t, args|
+    execute = args[:execute] == 'execute'
+
+    Tenant.safe_switch_each do |tenant|
+      puts "\nProcessing tenant #{tenant.host} \n\n"
+
+      data_units = ReportBuilder::PublishedGraphDataUnit.where("data->'phase' IS NOT NULL AND data->'phase'->'vote_term' IS NULL")
+
+      data_units.each do |data_unit|
+        builder = ReportBuilder::ReportPublisher.new(data_unit.report)
```

Comment:
@luucvanderzee Only thing I'm not sure about, does this risk publishing anything that currently is not yet published?

I don't think it does, as PublishedGraphDataUnit records only exist once something is published, but want to be sure.

Reply:
I think it should only republish stuff that is already published indeed.